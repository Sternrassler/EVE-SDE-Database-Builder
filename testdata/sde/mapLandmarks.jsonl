{"landmarkID":60000001,"landmarkName":"EVE Gate","description":"The EVE Gate is a collapsed wormhole.","locationID":30000001,"x":0.0,"y":0.0,"z":0.0,"iconID":null,"radius":1.0e10}
{"landmarkID":60000002,"landmarkName":"Jita Trade Hub","description":"The largest trade hub in New Eden.","locationID":30000142,"x":0.0,"y":0.0,"z":0.0,"iconID":null,"radius":5.0e9}
{"landmarkID":60000003,"landmarkName":"Null Sec Monument","description":"Monument to capsuleer conflicts.","locationID":30000003,"x":0.0,"y":0.0,"z":0.0,"iconID":null,"radius":2.0e9}
