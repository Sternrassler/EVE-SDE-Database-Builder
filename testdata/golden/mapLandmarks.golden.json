[
  {
    "landmarkID": 60000001,
    "landmarkName": "EVE Gate",
    "description": "The EVE Gate is a collapsed wormhole.",
    "locationID": 30000001,
    "x": 0,
    "y": 0,
    "z": 0,
    "iconID": null
  },
  {
    "landmarkID": 60000002,
    "landmarkName": "Jita Trade Hub",
    "description": "The largest trade hub in New Eden.",
    "locationID": 30000142,
    "x": 0,
    "y": 0,
    "z": 0,
    "iconID": null
  },
  {
    "landmarkID": 60000003,
    "landmarkName": "Null Sec Monument",
    "description": "Monument to capsuleer conflicts.",
    "locationID": 30000003,
    "x": 0,
    "y": 0,
    "z": 0,
    "iconID": null
  }
]